app/src/main/java/org/squidfish/tuplenback/data/room/GameStatsDao.kt

```diff
@@ -0,0 +1,35 @@
+package org.squidfish.tuplenback.data.room
+
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.Query
+import org.squidfish.tuplenback.games.Game
+
+@Dao
+interface GameStatsDao {
+    @Query(
+        """
+        SELECT * FROM gamestatsdata
+        WHERE gameType = :gameType
+        ORDER BY gameEndTime DESC
+        LIMIT :amount
+    """,
+    )
```

Comment:
The `"""` can be pulled to be on the same lines as the braces (I don't think that violates the formatting the linter sets but correct me if I'm wrong)